src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -92,6 +132,104 @@ class Dashboard extends PureComponent {
+                          onChange={event =>
+                            this.handleChangeRange(event.target.value)
+                          }
+                        >
+                          <MenuItem value={1}>1 hour</MenuItem>
+                          <MenuItem value={2}>2 hours</MenuItem>
+                          <MenuItem value={3}>6 hours</MenuItem>
+                          <MenuItem value={4}>24 hours</MenuItem>
+                          <MenuItem value={5}>1 week</MenuItem>
+                          <MenuItem value={6}>1 month</MenuItem>
+                          <MenuItem value={7}>6 months</MenuItem>
+                          <MenuItem value={8}>1 year</MenuItem>
+                        </Select>
+                      </FormControl>
+                    </Paper>
+                  </Grid>
+                  <Grid item xs={12} sm={12} md={6}></Grid>
+                </Grid>
+              </Fragment>
+            )}
+            {value === 2 && (
+              <Fragment>
+                <Grid container spacing={24}>
+                  <Grid item xs={12} sm={12} md={6}>
+                    <Typography variant="subtitle1" color="primary" paragraph>
+                      Sustaingineering is a student engineering design team that
+                      designs, develops and deploys sustainable technology
+                      solutions for renewable energy applications in remote and
+                      developing communities. Our goal is to create power
+                      solutions to address the global challenge of climate
+                      change and to improve the quality of life of the people
+                      living in these communities.
+                    </Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={12} md={6}>
+                    <Typography variant="subtitle1" color="primary" paragraph>
+                      Sustaingineering is a student engineering design team that
+                      designs, develops and deploys sustainable technology
+                      solutions for renewable energy applications in remote and
+                      developing communities. Our goal is to create power
+                      solutions to address the global challenge of climate
+                      change and to improve the quality of life of the people
+                      living in these communities.
+                    </Typography>
+                  </Grid>
+                </Grid>
+              </Fragment>
+            )}
+            {value === 3 && (
+              <Fragment>
+                <Grid
+                  container
+                  spacing={24}
+                  direction="column"
+                  alignItems="center"
+                >
+                  <Grid item>
+                    <div>
+                      <h1>Upload your csv file</h1>
+
+                      <UploadData
+                        keys={keys}
```

Comment:
Similarly, we shouldn't need to pass this as a prop (because the keys should be from the csv header).